assets/bare_pytorch_right_part.py

```diff
@@ -0,0 +1,55 @@
+model = Net()
```

Comment:
Are these python files needed? If not we should remove it from this PR

Reply:
Idea to keep them is to trace the sources of how side-by-side image has been created in case if we would like to change something (a minor change) without rewriting everything...

Reply:
I think wiki is a better idea as you mentioned on slack :)